.travis.yml

```diff
@@ -4,15 +4,17 @@ sudo: false
 
 matrix:
   include:
-    - node_js: 1
+    - node_js: node
       script:
         - npm run-script danger
         - npm run-script lint
         - npm run-script test-with-coverage
+        - if [[ $TRAVIS_NODE_VERSION == "8" ]]; then npm install dtslint; npm run dtslint; fi
```

Comment:
Realizing that, since this script is just running on the latest node (which'll be >= 8), we can probably just remove the if statement entirely and directly `npm install`. Seem reasonable?